httpc.go

```
@@ -803,7 +759,7 @@ func (c *Client) retryRoundTripper(next http.RoundTripper) http.RoundTripper {
 
 			// 在重试前，确保关闭当前失败的响应体以复用连接
 			if resp != nil && resp.Body != nil {
-				io.Copy(io.Discard, resp.Body)
+				copyb.Copy(io.Discard, resp.Body)
```

golangci-lint (1.64.8)

Error return value of `copyb.Copy` is not checked

(errcheck)